samples/charts/financial-chart/annotations/Pages/FinancialChartAnnotations.razor

```diff
@@ -0,0 +1,182 @@
+@using Infragistics.Blazor.Controls
+@inject IInfragisticsBlazor InfragisticsBlazor;
+@inject IJSRuntime JSRuntime;
+@inject Samples.Services.StocksUtility StocksUtility;
+@inject Samples.Services.Stock Stock;
+
+@* NOTE use this HTML code to design a sample *@
+<div class="igContainer">
+    <div class="igOptions">
+        <label class="igOptions-label">Annotations: </label>
+        <label class="igOptions-item">
+            <input type="checkbox"
+                   checked=@CrosshairsVisible
+                   onchange=OnCrosshairsVisible /> Crosshair
+        </label>
+        <label className="igOptions-item">
+            <input type="checkbox"
+                   checked=@CalloutsVisible
+                   onchange=OnCalloutsVisible /> Callouts
+        </label>
+        <label className="igOptions-item">
+            <input type="checkbox"
+                   checked=@FinalValuesVisible
+                   onchange=OnFinalValuesVisible /> Final Values
+        </label>
+        <label className="igOptions-item">
+            <input type="checkbox"
+                   checked=@MarkersVisible
+                   onchange=OnMarkersVisible /> Markers
+        </label>
+    </div>
+
+    <div class="igComponent">
+
+        @* NOTE making sure data is loaded before we can render it *@
+        @if (DataSource != null)
+        {
+            <FinancialChart Width="100%"
+                            Height="100%"
+                            ChartType=FinancialChartType.Line
+                            Thickness=2
+                            ExcludedProperties="ExcludedProperties"
+                            XAxisMode="XAxisMode"
+                            YAxisMode="YAxisMode"
+                            DataSource="DataSource"
+                            
+                            CalloutsVisible="CalloutsVisible"
+                            CalloutsXMemberPath="Index"
+                            CalloutsYMemberPath="Value"
+                            CalloutsLabelMemberPath="Info"
+                            CalloutsContentMemberPath="Info"
+                            CrosshairsSnapToData=false
+                            CrosshairsDisplayMode="CrosshairsMode"
+                            CrosshairsAnnotationEnabled="CrosshairsVisible"
+                            FinalValueAnnotationsVisible="FinalValuesVisible" />
+        }
+
+    </div>
+</div>
+
+@* NOTE you can change this C# code to affect behaviour of this page: *@
+@code {
+
+    public string[] ExcludedProperties;
+    public FinancialChartXAxisMode XAxisMode;
+    public FinancialChartYAxisMode YAxisMode;
+    Samples.Services.Stock[] DataSource;
+    public Boolean CalloutsVisible;
+    public String Index;
+    public string Value;
+    public CrosshairsDisplayMode CrosshairsMode;
+    public Boolean CrosshairsVisible;
+    public Boolean FinalValuesVisible;
+    public Boolean MarkersVisible;
+    public MarkerType MarkerTypes;
```

Comment:
you need to initialize all above properties (except of DataSource) on the same line where they are declared.